Source/OxyPlot/Series/BoxPlotSeries.cs

```diff
@@ -94,6 +97,18 @@ public BoxPlotSeries()
         public double MedianThickness { get; set; }
 
         /// <summary>
+        /// Gets or sets the size of the mean point.
+        /// </summary>
+        /// <remarks>This property is only used when ShowMeanAsDot = true.</remarks>
+        public double MeanPointSize { get; set; }
```

Comment:
Are we feature bloating this class now? 
The next feature requests will be custom median/mean marker shapes, fill colors etc.

Is line style really needed? Do we need other marker styles than line and dot? 
Shouldn't MedianMarkerThickness be renamed to MedianMarkerSize? 
Can all markers (outliers, median, mean) have the same marker sizes?